oeps/oep-0001.rst

```diff
@@ -300,111 +368,84 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
-the number of the OEP that it rendered obsolete.
++-----------------+-------------------------------------------+
+| OEP             | OEP-XXXX-<brief-type>-<brief-title>       |
++-----------------+-------------------------------------------+
+| Title           | <OEP title>                               |
++-----------------+-------------------------------------------+
+| Last Modified   | <date string, in YYYY-MM-DD format>       |
++-----------------+-------------------------------------------+
+| Authors         | <list of authors' real names and          |
+|                 | optionally, email addresses>              |
++-----------------+-------------------------------------------+
+| Arbiter         | <Arbiter's real name and email address>   |
++-----------------+-------------------------------------------+
+| Status          | <Draft | Under Review | Deferred |        |
+|                 | Accepted | Rejected | Withdrawn |         |
+|                 | Final | Replaced>                         |
++-----------------+-------------------------------------------+
+| Type            | <Architecture | Best Practice | Process>  |
++-----------------+-------------------------------------------+
+| Created         | <date created on, in YYYY-MM-DD format>   |
++-----------------+-------------------------------------------+
+| `Review Period` | <start - target end dates for review>     |
++-----------------+-------------------------------------------+
+| `Resolution`    | <links to any discussions where the final |
+|                 | status was decided>                       |
++-----------------+-------------------------------------------+
+| `Replaces`      | <OEP number>                              |
++-----------------+-------------------------------------------+
+| `Replaced-By`   | <OEP number>                              |
++-----------------+-------------------------------------------+
+| `References`    | <links to any other relevant discussions  |
+|                 | or relevant related materials>            |
++-----------------+-------------------------------------------+
+
+* The **Authors** header lists the names, and optionally the email addresses, of
+  all the authors/owners of the OEP. The format of the Authors header value must be
+  ``Random J. User <address@dom.ain>`` if the email address is included, or
+  ``Random J. User`` if the address is not given. If there are multiple authors,
+  their names and addresses should appear in a comma separated list.
+
+* The **Arbiter** field is used to record who has the authority to make the final
+  decision to approve or reject the OEP.
+
+* The **Type** header specifies the type of OEP: Architecture, Best Practice, or
+  Process.
+
+* The **Created** header records the date that the pull request for the OEP was
+  opened. It should be in YYYY-MM-DD format, e.g. 2016-04-21.
+
+* The **Review Period** header specifes the target dates for reviewing the OEP, as
```

Comment:
Would it be helpful to have a PR template (automated or otherwise) that includes this info at the top of the description?